Cargo.toml

```diff
@@ -15,88 +16,105 @@ opt-level = 0
 overflow-checks = true
 panic = "unwind"
 
+[patch."https://github.com/input-output-hk/midnight-ledger-prototype"]
```

Comment:
Are these private packages?
Are they going to introduce issues with CI and build?

Reply:
These are actually renamed repos (they were renamed to midnightntwrk). Some of their code still references IOHK so they require a patch. Once they open-source I don't imagine there will be any CI issues.